### ISSUE TITLE
NEED HELP?

### ISSUE THREAD

**Issue by SwathingBlock** [Mpstark/articulate#100]

**Comment by SwathingBlock** [comment]
Hi, Sorry for posting this here, I just wanted to say that I watched a view from you from a long time ago abut articulate and I love getting into new projects, I'm a research and development engineer for a home automation company and one of my own driver for that company was actually a voice recognition system for controlling a whole house so maybe some of my experience can help you and I am specialized in Java and C  sharp so yeah, let me know if you're looking for help :)

**Comment by notheotherben** [comment]
Hi there @SwathingBlock,

We're certainly open to anybody who wants to adopt this project and start extending it to support new features, improving the features that are already there or anything else for that matter. Unfortunately neither @Mpstark nor myself have had much time to invest in this in years, however please don't let that discourage you from contributing if you'd like to.

The source code should all continue to compile, however I haven't tested it in about 6 months at this point. There's also a nice list of open issues which describe some feature requests that people have and as I'm sure you've noticed, one of the biggest problems is the fact that the default ArmA 3 keybinds are now different to the ones provided by Articulate. There was also discussion of making it easily possible for people to customize their commands and add new ones, however I believe that effort stalled and no progress has been made since.

Thank you for the offer and let me know if I can help you out in any way,
Ben

**Comment by SwathingBlock** [comment]
Hi,


Yes I noticed that it hasn't been worked on for a while now, if it's something you would like to extend I would definitely be interested to help, I get into all sorts of projects and I think your project or at least the concept is very good and has some good potential, bare in mind, I'm not just thinking about Arma 3, if with that aim, this could possibly be a project that can extend and cover other games even though I can't say I have properly looked through the code yet to see how "Arma specific" this actually is in terms of integration. A good voice control software is something that people quite enjoy and look for in many other games, I myself, for year was addicted to Flight Simulator, where voice control is an amazing feature that a lot of players and even real life pilots, this is just an example, there are a few out there where you could potentially provide a very nice tool people would enjoy and even pay for if extended.


I'll play around with the project and look through the code when I have some time and thanks for your quick reply.


Thanks


________________________________
De: Benjamin Pannell <notifications@github.com>
Enviado: 2 de outubro de 2018 08:24
Para: Mpstark/articulate
Cc: SwathingBlock; Mention
Assunto: Re: [Mpstark/articulate] NEED HELP? (#100)


Hi there @SwathingBlock<https://github.com/SwathingBlock>,

We're certainly open to anybody who wants to adopt this project and start extending it to support new features, improving the features that are already there or anything else for that matter. Unfortunately neither @Mpstark<https://github.com/Mpstark> nor myself have had much time to invest in this in years, however please don't let that discourage you from contributing if you'd like to.

The source code should all continue to compile, however I haven't tested it in about 6 months at this point. There's also a nice list of open issues which describe some feature requests that people have and as I'm sure you've noticed, one of the biggest problems is the fact that the default ArmA 3 keybinds are now different to the ones provided by Articulate. There was also discussion of making it easily possible for people to customize their commands and add new ones, however I believe that effort stalled and no progress has been made since.

Thank you for the offer and let me know if I can help you out in any way,
Ben

**Comment by Shigbeard** [comment]
If you are going to pick up the project, may I ask that you work to incorporate Google's voice API? It should be possible to do, and google has a far more accurate speech detection engine.

**Comment by SwathingBlock** [comment]
Yeah, it shouldn't be a big problem to do so, I have personally built a driver for my automation company, to control a house using voice recognition and we also built a solution using Amazon's Alexa, which is similar to Google's . The only disadvantage I find with Siri, Alexa, Google Assistant ... is that, you always need to initialize the session, by saying something like "Alexa, ask (skill name ) to ...." or "Ok google ..." which becomes very very annoying since you, as a developer, are not allowed to remain a session open for more than a few seconds, 8 seconds or so. I just came back from some holidays so I in the next few days at some point I will be looking at the code you have done so far and see what pops to mind.


Thanks


________________________________
De: Shigbeard_ <notifications@github.com>
Enviado: 15 de outubro de 2018 13:29
Para: Mpstark/articulate
Cc: SwathingBlock; Mention
Assunto: Re: [Mpstark/articulate] NEED HELP? (#100)


If you are going to pick up the project, may I ask that you work to incorporate Google's voice API? It should be possible to do, and google has a far more accurate speech detection engine.
